### PR TITLE
Feature/readme

### PR DESCRIPTION
**Pull request by jeovazero**
For update the root readme call the readme:update script

### CODE REVIEW

File: scripts/createComponent.js [hunk 1]
```diff
@@ -1,14 +1,24 @@
...
 const COMPONENT_BASE_NEW_VERSION = 'lernaversion'
 const COMPONENT_BASE_NAME = 'Component'
 const COMPONENT_BASE_PATH = './templates/component'
+
+if (process.argv.length < 3)
+  return console.log('\x1b[31m', '[Bolt] ❌ Name of component is required and must be in CamelCase style ❌')
+
 const COMPONENT_NAME = process.argv[2]
+
+if (!isCamelCase(COMPONENT_NAME)) {
```
Comment (guilhermedecampo) [inline scripts/createComponent.js]: Nice one

File: scripts/createComponent.js [hunk 2]
```diff
@@ -1,14 +1,24 @@
 const fs = require('fs')
 const path = require('path')
-const lernaInfo = require('../../lerna.json')
```
Comment (guilhermedecampo) [inline scripts/createComponent.js]: Thanks!

File: scripts/updateRootReadme.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  `| [\`bolt-${packageName}\`](/packages/${packageName})` +
+  `| [![npm version](https://badge.fury.io/js/%40pismo%2Fbolt-${packageName}.svg)](https://badge.fury.io/js/%40pismo%2Fbolt-${packageName})` +
+  `| [${toCamelCase(packageName)} Documentation](https://bolt.pismolabs.io/packages-core-components-${packageName}) |`
+
+const table = ['| Package | Version | Documentation |', '| - | - | - |', ...packagesNames.map(pkg => lineTemplate(pkg))]
+
+const newListComponentsBlock = [`## Components`, '', ...table, '']
+
+const newReadme = [
+  ...listLines.slice(0, startComponentsBlock),
```
Comment (guilhermedecampo) [inline scripts/updateRootReadme.js]: Spread is awesome

**Comment by jeovazero** [comment]
The `readme:update` script in action:

readme

**Pull request merged**